src/TestReports.jl

```diff
@@ -7,9 +7,12 @@ using EzXML
 import Test: AbstractTestSet, DefaultTestSet, record, finish, get_testset_depth, get_testset
 import Test: Result, Fail, Broken, Pass, Error, scrub_backtrace
 
-export ReportingTestSet, any_problems, report, display_reporting_testset
+export ReportingTestSet, any_problems, report, display_reporting_testset,
+    @recordproperty
 
 include("./testsets.jl")
 include("to_xml.jl")
 include("runner.jl")
+include("macros.jl")
```

Comment:
Macros is not a good name for this file.
it tells me about implementation, rather than features

Reply:
Good point!